Documentation/DeveloperGuide/README.md

```diff
@@ -8,14 +8,22 @@ Follow these steps to run the addon directly from source so that your changes wi
 
 # Running on windows
```

Comment:
it can only run on windows so we don't need this section, but rather a note about that